oneflow/api/python/framework/tensor_functions.cpp

```diff
@@ -273,6 +282,73 @@ static PyObject* PyTensorObject_reshape_as(PyObject* self, PyObject* args, PyObj
   END_HANDLE_ERRORS
 }
 
+static PyObject* PyTensorObject_view(PyObject* self, PyObject* args) {
+  HANDLE_ERRORS
+  PyObject* shape = args;
+  if (PyTuple_Size(args) == 1) {
+    PyObject* item = PyTuple_GetItem(args, 0);
+    if (!PyLong_Check(item)) { shape = item; }
+  }
+  CHECK_OR_THROW(functional::PyLongSequenceCheck(shape))
+      << Error::TypeError() << "view() received an invalid combination of arguments - got ("
+      << Py_TYPE(shape)->tp_name << "), but expected tuple of ints size.";
+
+  const auto& dims = functional::PyUnpackLongSequence<int64_t>(shape);
+  DimVector dim(dims.begin(), dims.end());
+  return PyTensor_New(ASSERT_PTR(functional::View(PyTensor_Unpack(self), Shape(dim))));
+  END_HANDLE_ERRORS
+}
+
+static PyObject* PyTensorObject_view_as(PyObject* self, PyObject* args, PyObject* kwargs) {
+  HANDLE_ERRORS
+  auto tensor = PyTensor_Unpack(self);
+  PyObject* other = NULL;
+  static const char* keywords[2] = {"other", NULL};
+  if (!PyArg_ParseTupleAndKeywords(args, kwargs, "O|:view_as", const_cast<char**>(keywords),
+                                   &other)) {
+    return NULL;
+  }
+  return PyTensor_New(ASSERT_PTR(functional::View(tensor, *PyTensor_Unpack(other)->shape())));
+  END_HANDLE_ERRORS
+}
+
+static PyObject* PyTensorObject_permute(PyObject* self, PyObject* args, PyObject* kwargs) {
+  HANDLE_ERRORS
+  PyObject* dims = args;
+  if (PyTuple_Size(args) == 0) {
+    // keyword parameter
+    static const char* keywords[2] = {"dims", NULL};
+    if (!PyArg_ParseTupleAndKeywords(args, kwargs, "O|:permute", const_cast<char**>(keywords),
+                                     &dims)) {
+      return NULL;
+    }
+  } else if (PyTuple_Size(args) == 1) {
+    // positional parameter
+    PyObject* item = PyTuple_GetItem(args, 0);
+    if (!PyLong_Check(item)) { dims = item; }
+  }
+  CHECK_OR_THROW(functional::PyLongSequenceCheck(dims))
+      << Error::TypeError() << "permute(): argument 'dims' must be tuple of ints, but found "
+      << Py_TYPE(dims)->tp_name;
+  const auto& dims_vec = functional::PyUnpackLongSequence<int32_t>(dims);
+  return PyTensor_New(ASSERT_PTR(functional::Permute(PyTensor_Unpack(self), dims_vec)));
+  END_HANDLE_ERRORS
+}
+
+static PyObject* PyTensorObject_transpose(PyObject* self, PyObject* args, PyObject* kwargs) {
+  HANDLE_ERRORS
+  auto tensor = PyTensor_Unpack(self);
+  int dim0 = 0;
+  int dim1 = 0;
+  static const char* keywords[3] = {"dim0", "dim1", NULL};
+  if (!PyArg_ParseTupleAndKeywords(args, kwargs, "i|i:transpose", const_cast<char**>(keywords),
```

Comment:
```suggestion
  if (!PyArg_ParseTupleAndKeywords(args, kwargs, "ii:transpose", const_cast<char**>(keywords),
```
我看了一下pytorch和之前的oneflow，这里的dim1不是optional的
```
In [8]: torch.randn(3, 3).transpose(1)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Input In [8], in <cell line: 1>()
----> 1 torch.randn(3, 3).transpose(1)

TypeError: transpose() received an invalid combination of arguments - got (int), but expected one of:
 * (int dim0, int dim1)
 * (name dim0, name dim1)

In [22]: flow.rand(3, 3).transpose(0)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Input In [22], in <module>
----> 1 flow.rand(3, 3).transpose(0)

TypeError: _transpose() missing 1 required positional argument: 'dim1'
```

Reply:
好的，已修改